code/modules/culture_descriptor/faction/factions_human.dm

```diff
@@ -37,17 +37,14 @@
 	economic_power = 1.2
 	subversive_potential = 35
 
-/decl/cultural_info/faction/gcc
-	name = FACTION_INDIE_CONFED
-	description = "The Independent Colonial Confederation of Gilgamesh, commonly referred to as the Gilgamesh Colonial Confederation (GCC), is a regional power in human space,\
-	bordered by the Sol Central Government towards the galactic centre. The capital of the Confederation is the city of Ameranth on the planet of Terra \
-	in the Gilgamesh system. Externally heavily militant and isolationist, the GCC, internally, is heavily libertarian, with \
-	a strong focus on independent planetary government with the GCC itself only handling defence, foreign relations and some intergalactic trade. \
-	Increasingly anti non-human, the GCC is in a cold war with the Sol Central Government following  the Gaia Conflict, a large \
-	scale conflict between the SCG and GCC that saw a great deal of death and destruction. In Sol space \"Indies\" as they are known are \
-	heavily mistrusted and in some instances, met with downright hostility. Today the GCC continues to be an economic powerhouse following \
-	a period of steady economic recovery, with most of its exports being manufactured goods, primarily armaments."
-	economic_power = 0.9
+/decl/cultural_info/faction/terragov
+	name = FACTION_TERRAGOV
+	description = "The Terran Government, commonly referred to as TerraGov or TG, is a second most powerful human government in the universe. \
+	As opposed to SolGov, TerraGov's ideals are those of progressing humanity at any cost, especially at the cost of alien civilizations. \
+	Terran Government prohibits any political parties that oppose the Terran Constitution, and as such there is only so much factions within the senate \
+	and each would rephrase 'progress of humanity' as they see fit, ranging from a more peaceful approach of co-existence with non-hostile alien \
```

Comment:
Either:
[...] only so much factions within the senate can do, where each would rephrase [...]
Or:
[...} only so many factions within the senate and each would [...]